osu.Game/Online/Leaderboards/LeaderboardScore.cs

```diff
@@ -96,6 +93,8 @@ private void load(IAPIProvider api, OsuColour colour, ScoreManager scoreManager)
 
             statisticsLabels = GetStatistics(Score).Select(s => new ScoreComponentLabel(s)).ToList();
 
+            this.scoreManager = scoreManager;
```

Comment:
If you're going to do this then you may as well move `ScoreManager` out from the BDL param to a `[Resolved]` field.